docs/pages/client_usage.mdx

```diff
@@ -0,0 +1,237 @@
+
+Some origins do not support authentication though CILogon. In this case, users must supply their own JWT that's signed by the origin.
+
+> We are working hard to document how to generate a token from protected origins, before it’s ready. If you have any questions or need help getting a token from the origin, please email us at [help@pelicanplatform.com](mailto:help@pelicanplatform.com) or open a discussion on our [GitHub discussions page](https://github.com/PelicanPlatform/pelican/discussions) and we are happy to assist you.
+
+Contact your federation administrator to get a token for downloading a protected object. For OSDF users, contact [support@osg-htc.org](mailto:support@osg-htc.org) for help.
+
+Once you have the token, copy and paste it into a file on your machine, and pass the path to the file containing the token to the Pelican client with `-t` flag.
+```bash
+# Pass the path to the file containing the token using -t flag
+pelican object get pelican://<federation-url></namespace-prefix></path/to/file> </local/path/to/file> -t </path/to/token/file>
+```
+
+For example, if a token is saved in a file named `my-token`, it can be used to get the object `/ospool/PROTECTED/auth-test.txt` by running:
+```bash
+# Get the object with the path to token file provided via -t flag
+$ pelican object get -f https://osg-htc.org /ospool/PROTECTED/auth-test.txt downloaded-auth-test.txt -t my-token
+```
+
+## PUT an object to Your Federation
+Another powerful Pelican client command is the `pelican object put` command. This command does a simple PUT request to add your object to your federation. Since the vast majority of federations will not allow public writes, we will assume for the example that we will need a token to perform these requests. Here is how you can use `pelican object put`:
+
+```bash
+pelican object put <path/to/local/file> pelican://<federation-url></namespace-prefix></path/to/file> -t </path/to/token/file>
+```
+> **Note:** you can also specify the federation url here with the `-f` flag, just be sure not to include it in the request URL as the host name if you decide to do so.
+
+## Pelican Object Copy
+As well with Pelican's `object get` and `object put` commands, there is also an older command called `pelican object copy`. This functions the same as the `pelican object get/put` except that it works for both gets and puts. For example, to do an `object get`:
+```bash
+pelican object copy pelican://<federation-url></namespace-prefix></path/to/file> <local/path/to/file>
+```
+and to do an `object put`:
+
+```bash
+pelican object copy <path/to/local/file> pelican://<federation-url></namespace-prefix></path/to/destination> -t </path/to/token/file>
+```
+
+## Utilizing Queries with your URL
+The Pelican client has the ability to select various amount of queries specified in the remote path to an object. The queries the current Pelican client accepts are: ?pack, ?recursive, and ?directread
+
+### ?pack Query
```

Comment:
Instead of beginning a header with the query, let's say something about what the section tells you how to do. For this one, maybe `Packing Objects with the ?pack Query`. The next header could be `Recursive Downloads with the ?recursive Query`